src/jobs/membership.ts

```diff
@@ -0,0 +1,51 @@
+import cron from 'node-cron';
+import User from '../models/user';
+import { MEMBERSHIP_STATUS } from '../types/user';
+import {
+  GoogleSpreadsheet,
+  WorksheetBasicProperties,
+} from 'google-spreadsheet';
+import GoogleCreds from '../config/acs-web-5aaaf-768e669e2d39.json';
+
+const getTerm = () => {
+  const date = new Date();
+  const month = date.getMonth();
+  if (month === 1) {
+    return 'Fall' + (date.getFullYear() - 1);
```

Comment:
Wouldn't `month === 1` correspond to winter? And `date.getMonth()` is zero-based so 1 is February

Reply:
I think we run the job at the end of the term. 
1st of Jan runs for Fall
1st of May runs for Winter
1st of Sept runs for Spring
lmk if this is okay or if I should leave a comment in the code

Reply:
Fine by me. Still concerned about `month === 1` being February tho

Reply:
¡¡¡!!! 

Thanks for the catch!!!